man-roxygen/DataImport.yml

```diff
@@ -0,0 +1,29 @@
+extract:
+  short: >-
+    Run the extract stage of the data import
+
+transform:
+  short: >-
+    Run the transform stage of the data import
+
+load:
+  short: >-
+    Run the load stage of the data import
+  params:
+    comment: >-
+      Optional comment which will be persisted in the log of the import run
+      in the database.
+    dry_run: >-
+      Whether to run in dry run mode. If TRUE then any database changes will
+      be rolled back. Defaults to FALSE.
+    force: >-
+      If TRUE then checks that repo is up to date with git remote will be
+      skipped. Defautls to FALSE.
```

Comment:
`Defautls` - take a look at `spelling::spell_check_package()` and see what it turns up.  I find it works best by:

* add `Language: en-GB` to your DESCRIPTION
* put false positives into a file `inst/WORDLIST` (one word per line)

Reply:
Thanks, I did not know about this. I've fixed all of the stuff it flagged up.